.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
 
 node_js:
-  - 0.10
+  - "4.0"
```

Comment:
http://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Choosing-Node-versions-to-test-against